etc/library.sh

```diff
@@ -230,6 +230,33 @@ function persistent_cfg()
   ln -s "$DST" "$SRC"
 }
 
+function is_more_recent_than()
+{
+  local version_A="$1"
+  local version_B="$2"
+ 
+  MAJOR_A=$( echo ${version_A} | cut -d. -f1 )
```

Comment:
```suggestion
  local major_a=$( echo ${version_A} | cut -d. -f1 )
```

(there's more in this function)